custom_domains/getanid/main.tf

```diff
@@ -0,0 +1,56 @@
+module "domains" {
+  source              = "git::https://github.com/DFE-Digital/terraform-modules.git//domains/environment_domains?ref=add-custom-domain-fd"
+  zone                = var.zone
+  front_door_name     = var.front_door_name
+  resource_group_name = var.resource_group_name
+  domains             = var.domains
+  environment         = var.environment_short
+  host_name           = data.azurerm_linux_web_app.app.default_hostname
+
+}
+
+data "azurerm_cdn_frontdoor_profile" "main" {
+  name                = var.front_door_name
+  resource_group_name = var.resource_group_name
+}
+
+data "azurerm_linux_web_app" "app" {
+  provider            = azurerm.app_subcription
+  name                = var.app_name
+  resource_group_name = var.app_resource_group_name
+}
+
+data "azurerm_dns_zone" "main" {
+  name                = var.zone
+  resource_group_name = var.resource_group_name
+}
+
+resource "azurerm_cdn_frontdoor_rule_set" "ruleset" {
+  name                     = var.environment_short
+  cdn_frontdoor_profile_id = data.azurerm_cdn_frontdoor_profile.main.id
+}
+
+resource "azurerm_cdn_frontdoor_rule" "rule" {
+  count = var.environment_short == "prod" ? 1 : 0
```

Comment:
We should not hardcode the environment name here but rather add a variable to the tfvars.
For example: `redirect_rules = [ {source = "www.teaching-identity.education.gov.uk", dest =  "teaching-identity.education.gov.uk"} ]`
And use a `for_each` here